### PR TITLE
toolchain foss/2018.08 with GCC/8.2.0

### PR DESCRIPTION
**Pull request by pescobar**
this still doesn't work . When building OpenMPI I get this error

```
  CC       dss/dss_open_close.lo
  CCLD     libopen-pal.la
/bin/ld.gold: error: /scicore/home/scicore/easybuild/test-installs/apps/GCCcore/8.2.0/lib/gcc/x86_64-pc-linux-gnu/8.2.0/crtbeginS.o: unsupported reloc 42 against global symbol _ITM_deregisterTMCloneTable
/bin/ld.gold: error: /scicore/home/scicore/easybuild/test-installs/apps/GCCcore/8.2.0/lib/gcc/x86_64-pc-linux-gnu/8.2.0/crtbeginS.o: unsupported reloc 42 against global symbol _ITM_registerTMCloneTable
/scicore/home/scicore/easybuild/test-installs/apps/GCCcore/8.2.0/lib/gcc/x86_64-pc-linux-gnu/8.2.0/crtbeginS.o(.text+0x16): error: unsupported reloc 42
/scicore/home/scicore/easybuild/test-installs/apps/GCCcore/8.2.0/lib/gcc/x86_64-pc-linux-gnu/8.2.0/crtbeginS.o(.text+0x57): error: unsupported reloc 42
collect2: error: ld returned 1 exit status
make[2]: *** [libopen-pal.la] Error 1
make[2]: Leaving directory `/scratch/easybuild-user/builds/OpenMPI/3.1.2/GCCcore-8.2.0/openmpi-3.1.2/opal'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/scratch/easybuild-user/builds/OpenMPI/3.1.2/GCCcore-8.2.0/openmpi-3.1.2/opal'
make: *** [all-recursive] Error 1
 (at easybuild/tools/run.py:501 in parse_cmd_output)
== 2018-10-09 18:34:16,129 easyblock.py:2851 WARNING build failed (first 300 chars): cmd " make -j 1 " exited with exit code 2 and output:
Making all in config
make[1]: Entering directory `/scratch/easybuild-user/builds/OpenMPI/3.1.2/GCCcore-8.2.0/openmpi-3.1.2/config'
make[1]: Nothing to be done for `all'.
make[1]: Leaving directory `/scratch/easybuild-user/builds/OpenMPI/3.1.2/GCC
```

### CODE REVIEW

**Comment by boegel** [comment]
@pescobar You don't have a build dependency on `binutils` in the `OpenMPI` easyconfig (the tests should also point that out); that probably explain the build problem you're seeing...

**Comment by pescobar** [comment]
the error in the first comment is fixed by using toolchain `GCC/8.2.0-2.30` that includes `binutils/2.30` . Thanks @boegel !

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 33 out of 33 (25 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3b5b18e029b8c24aabf4f612bb47b2e9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 25 out of 25 (25 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/296484e50837d5b6ee277966a590f426 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 25 out of 25 (25 easyconfigs in this PR)
node2617.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6de54ada638f0237dd6da32bcda3a3b8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 25 out of 25 (25 easyconfigs in this PR)
node2060.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c1f498c1943600500b72becc2ab70f08 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 25 out of 25 (25 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/d1cead1c772da1a62864c2cb16417914 for a full test report.

File: easybuild/easyconfigs/n/numactl/numactl-2.0.11-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'numactl'
+version = '2.0.11'
```
Comment (boegel) [inline easybuild/easyconfigs/n/numactl/numactl-2.0.11-GCCcore-8.2.0.eb]: please update to 2.0.12 (see https://github.com/numactl/numactl/releases), and fix homepage too

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 27 out of 27 (27 easyconfigs in this PR)
node3167.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/222381f50f68ea5453b432538be57165 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 27 out of 27 (27 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/afb455009bf9e6552da5629e51e8324c for a full test report.

**Comment by pescobar** [comment]
my test build is running....

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 27 out of 27 (27 easyconfigs in this PR)
node2004.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/86f8c4665fbe481517ef303872da7c04 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 27 out of 27 (27 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6aff3ef04ca0208b04904a71f1784aa8 for a full test report.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 27 out of 27 (27 easyconfigs in this PR)
aw-4r13-n22 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/61a52daa1a28080137c0620e51bb2da4 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 27 out of 27 (27 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/5c78739f26410224957a376d89b2bac5 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @pescobar!

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 27 out of 27 (27 easyconfigs in this PR)
dagur - Linux debian 9.5, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.13
See https://gist.github.com/ca01142f40b505da34d173fc705e37e2 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 30 out of 30 (27 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/46c1525bcccbf9e6c724cda1e1c657fc for a full test report.

**Pull request merged**